gto-support-util/src/main/java/org/ccci/gto/android/common/util/content/Context.kt

```diff
@@ -4,16 +4,20 @@ import android.content.Context
 import android.content.res.Configuration
 import android.os.Build
 import android.os.LocaleList
-import org.ccci.gto.android.common.util.os.toTypedArray
+import org.ccci.gto.android.common.util.os.locales
 import java.util.Locale
 
+@OptIn(ExperimentalStdlibApi::class)
 fun Context.localize(vararg locales: Locale, includeExisting: Boolean = true): Context = when {
     locales.isEmpty() -> this
     Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1 -> createConfigurationContext(Configuration().apply {
         when {
             Build.VERSION.SDK_INT >= Build.VERSION_CODES.N -> setLocales(
                 LocaleList(
-                    *locales, *(if (includeExisting) resources.configuration.locales.toTypedArray() else emptyArray())
+                    *buildList<Locale> {
```

Comment:
Does Kotlin not have `Set`? That would keep unique values by default.

Reply:
I want them to maintain the order. I could have used a LinkedHashSet, but then I wouldn't have access to the `build* {}` constructor